tests/unit/htdocs/Frameworks/art/FunctionsUserTest.php

```
@@ -0,0 +1,137 @@
+    #[Test]
+    public function getIpDeprecationMentionsFunction(): void
+    {
+        $logger = \XoopsLogger::getInstance();
+        $countBefore = count($logger->deprecated);
+
+        @mod_getIP(true);
+
+        $lastMsg = $logger->deprecated[count($logger->deprecated) - 1];
+        $this->assertStringContainsString('mod_getIP', $lastMsg);
+        $this->assertStringContainsString('XoopsUserUtility::getIP', $lastMsg);
+    }
```

PHPMD (2.15.0)

[error] Remove error control operator '@' on line 61. (undefined)

(ErrorControlOperator)